app/presenters/tree_node/menu/node.rb

```diff
@@ -0,0 +1,40 @@
+module TreeNode
+  module Menu
+    class Node < TreeNode::Node
+      set_attribute(:no_click, true)
+      set_attribute(:icon, "pficon pficon-folder-close")
+
+      set_attribute(:checkable) { @options[:editable] }
+      set_attribute(:selected)  { select_state }
```

Comment:
I think this can be removed if you have a `set_attribute` for each subclass with the same body as `select_state`.